apps/core-frontend/tailwind.config.js

```diff
@@ -3,10 +3,8 @@ const defaultTheme = require("tailwindcss/defaultTheme");
 
 module.exports = {
   // mode: "jit",
-  purge: {
-    // tree shake tailwind css for production builds
-    content: ["./src/**/*.{ts,tsx}"],
-  },
+  // tree shake tailwind css for production builds
+  content: ["./src/**/*.{ts,tsx}"],
```

Comment:
Hm, curious about this change..

For tailwind 1.9.6 the docs have this: https://v1.tailwindcss.com/docs/controlling-file-size#purge-css-options

Reply:
Ahh, you're right! I forget `core-frontend` is still running Tailwind v1, will configure it back. It was giving me a warning when it was linting this repository to switch over to the v3 convention because the linter is on v3 configs